packages/coq-core/coq-core.8.18.0/opam

```diff
@@ -0,0 +1,69 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "The Coq Proof Assistant -- Core Binaries and Tools"
+description: """
+Coq is a formal proof management system. It provides
+a formal language to write mathematical definitions, executable
+algorithms and theorems together with an environment for
+semi-interactive development of machine-checked proofs.
+
+Typical applications include the certification of properties of
+programming languages (e.g. the CompCert compiler certification
+project, or the Bedrock verified low-level programming library), the
+formalization of mathematics (e.g. the full formalization of the
+Feit-Thompson theorem or homotopy type theory) and teaching.
+
+This package includes the Coq core binaries, plugins, and tools, but
+not the vernacular standard library.
+
+Note that in this setup, Coq needs to be started with the -boot and
+-noinit options, as will otherwise fail to find the regular Coq
+prelude, now living in the coq-stdlib package."""
+maintainer: ["The Coq development team <coqdev@inria.fr>"]
+authors: ["The Coq development team, INRIA, CNRS, and contributors"]
+license: "LGPL-2.1-only"
+homepage: "https://coq.inria.fr/"
+doc: "https://coq.github.io/doc/"
+bug-reports: "https://github.com/coq/coq/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.09.0"}
+  "ocamlfind" {>= "1.8.1"}
+  "zarith" {>= "1.11"}
+  "ounit2" {with-test}
+]
+conflicts: [
+  "coq"   { < "8.17" }
+  "ocaml-option-flambda"
```

Comment:
If it's true that 5.1 is looking better, we could just conflict with 5.0 instead of flambda cc @palmskog ?
```suggestion
  "ocaml" {>= "5.0" && < "5.1"}
```

Reply:
I have very informally benchmarked OCaml 5.1 rcs on Coq project, and the Coq *performance* there is good. However, I don't know if the issues with flambda shown in CI here persist or not, I haven't experienced anything like those myself.

Reply:
What I had originally in mind was `"ocaml-option-flambda" {= "5.0"}` since this is what the failing test is running.

To me the error has nothing to do with flambda nor ocaml 5, but well, if the opam repository maintainers don't want to merge a package which (apparently) breaks that combination, they clearly have the right to blacklist it.

We do test flambda in our CI and we have no problem with it, so the current change is too restrictive, IMO.

Reply:
```suggestion
```